### PR TITLE
Fix user_delete() return value when user_delete() is successful. issue#85

### PR DESCRIPTION
**Pull request by iwashi**
As I wrote issue #85, user_delete() should return true when it's successful.

### CODE REVIEW

File: classes/auth/login/ormauth.php [hunk 1]
```diff
@@ -498,7 +498,14 @@ public function delete_user($username)
 		// if it was found, delete it
 		if ($user)
 		{
-			return $user->delete();
+			if ($user->delete())
+			{
+				return true;
+			}
+			else
```
Comment (Art4) [inline classes/auth/login/ormauth.php]: This else statement is unnecessary.

**Comment by WanWizard** [comment]
It can even be condensed to

```
if ($user)
{
    return (bool) $user->delete();
}

return false;
```

**Comment by iwashi** [comment]
`return (bool) $user->delete();` looks good. :+1:

**Comment by WanWizard** [comment]
You update the PR, or I close it and change it myself?

**Comment by kenjis** [comment]
@WanWizard 
He updated this PR already. See "File changed"

**Comment by Art4** [comment]
@kenjis I think, @WanWizard wants to have the `else` statement removed. Find this:

``` php
if ($user)
{
    return (bool) $user->delete();
}
else
{
    return false;
}
```

and replace it with this:

``` php
if ($user)
{
    return (bool) $user->delete();
}

return false;
```

**Comment by WanWizard** [comment]
Exactly. The else is redundant there.

**Comment by iwashi** [comment]
Okay, I'm going to update PR from now.

**Comment by WanWizard** [comment]
Thanks.

**Pull request merged**